src/mame/drivers/cedar_magnet.cpp

```diff
@@ -1061,6 +1061,46 @@ ROM_START( mag_day )
 	ROM_LOAD( "adayinspace 31_3_87.img", 0x00000, 0xf0000, CRC(bc65302d) SHA1(6ace68a0b5f7a07a8f5c318c5359011074e7f2ec) )
 ROM_END
 
+// track 79 side 1 (file offset 0x0ee800) would not read, but it appears to be outside of the used data
+// this also has many scores stored on the disk at offset 0x01cc00, invalidating these does reset the
+// score table to 'EFO 100000' scores, but then it never writes new scores?
+ROM_START( mag_pdak )
+	BIOS_ROM
+
+	ROM_REGION( 0x100000, "flop:disk", ROMREGION_ERASE00 )
+	ROM_LOAD( "paris.dsk 31_3_87.img", 0x00000, 0xf0000, CRC(2c4ee9e1) SHA1(22c2b75c16aca95ecf2199451c1bd12dd3a3844c) )
+ROM_END
+
+/*
+The following tracks/sides failed to read (bad disk)
+
+track:68:0 (file offset:0x0cc000 - 0x0cd7ff)
+track:69:0 (file offset:0x0cf000 - 0x0d07ff)
+track:70:0 (file offset:0x0d2000 - 0x0d37ff)
+track:71:0 (file offset:0x0d5000 - 0x0d67ff)
+track:72:0 (file offset:0x0d8000 - 0x0d97ff)
+track:73:0 (file offset:0x0db000 - 0x0dc7ff)
+track:74:0 (file offset:0x0de000 - 0x0df7ff)
+track:75:0 (file offset:0x0e1000 - 0x0e27ff)
+track:76:0 (file offset:0x0e4000 - 0x0e57ff)
+
+these areas aren't read by the code that currently loads, but other areas also didn't read consistently
+
+the 3 dumps in the set below contain different reads of tracks 0-67
+
+*/
+
+ROM_START( mag_drac )
+	BIOS_ROM
+
+	ROM_REGION( 0x100000, "flop:disk", ROMREGION_ERASE00 )
+	ROM_LOAD( "drac.dsk", 0x00000, 0xf0000, CRC(2b5ca6f8) SHA1(063ea3b55bf95d05c866c0fcdb41c307c484a4f8) )
+	ROM_LOAD( "drac2.dsk", 0x00000, 0xf0000, CRC(cf6c1dd2) SHA1(7adb5146b050172090556927bf6d30ba8265107a) )
+	ROM_LOAD( "drac3.dsk", 0x00000, 0xf0000, CRC(7060e4a2) SHA1(b8e5437afff11d57a40c092d005d6b075819537a))
```

Comment:
These should definitely be marked `BAD_DUMP`.

Reply:
yeah, forgot about that